packages/cli/src/actions/start.ts

```diff
@@ -15,10 +16,18 @@ const start: StartHandler = async (filename: string, opts: any) => {
     spinner.fail('Please specify the config path');
     return process.exit(1);
   }
-  filename = path.isAbsolute(filename) ? filename : path.join(process.cwd(), filename);
-  if (!existsSync(filename)) {
-    spinner.fail(`${filename} not exists`);
-    return process.exit(1);
+  const urlObj = url.parse(filename);
+  if (urlObj.protocol) {
+    if (urlObj.protocol !== 'http:' && urlObj.protocol !== 'https:') {
+      spinner.fail(`protocol ${urlObj.protocol} is not supported`);
+      return process.exit(1);
+    }
+  } else {
+    filename = path.isAbsolute(filename) ? filename : path.join(process.cwd(), filename);
```

Comment:
We could convert it to the file-protocol.

Reply:
How about we refactoring this into file-upload.

Reply:
You added the extra check at daemon, so I think it's required at this PR :)